great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -200,7 +216,70 @@ def get_metrics(
             parameters=parameters,
         )
 
-        # Obtain enforce_numeric_metric from rule state (i.e., variables and parameters); from instance variable otherwise.
+        metric_values: Union[List[Any], np.ndarray] = []
+
+        metric_value: Union[Any, List[Any]]
```

Comment:
Can we get any more specific with this? By possibly being `Any`, it sort of negates to value of type hinting.

Reply:
@cdkini I am not sure how -- all I am trying to say is that the output of a metric can be a single quantity of any kind of a type, or a list thereof.  I think that in a future PR, we can create a special type for it, but for now, let us leave it.  I will add a note.  Thanks.